Documentation/Signoffs/SpeedometerSystem.md

```diff
@@ -0,0 +1,48 @@
+SPEEDOMETER SUBSYSTEM
+--------------------
+### Functions of the Subsystem
+The speedometer subsystem must receive output from the current driving simulation and display the speed the user is going. A microcontroller will be used to pull the speed data from the MATLAB code and display it on 4-digit a 7-segment display.
+### Specifications & Constraints
+-	Microcontroller must be compatible with MATLAB
+-	Must be able to display the speed at which the user is going in the simulator
+-	Shall connect to the PC running the simulation
+-	Shall display the speed in real-time
+-	Shall be powered by the PC	
+### Schematic
+##### 1. Pinouts
+
+![image](https://user-images.githubusercontent.com/117474540/202864081-4325652c-c3de-4a34-bf38-b649f1cd59cb.png)
+
+_Figure 1. Pinout of Arduino Uno_
+
+![image](https://user-images.githubusercontent.com/117474540/202864050-6e376c04-8fbd-40f2-bc74-d78c4ec03fb3.png)
+
+_Figure 2. Pinout of TM1637 4-digit 7-segment display_
+
+##### 2. Wiring Schematic
+
+![image](https://user-images.githubusercontent.com/117474540/202863610-1f4f66dd-5dce-49d9-b17f-d742b2c2202e.png)
+
+_Figure 3. Wiring diagram for the Arduino Uno and 4-digit 7-segment display_
+
+### Analysis
```

Comment:
Why are the selections being discussed? This area is where you show that the constraints will be met and the function achieved. If you find that you are not sure what target you are shooting at, the constraints are incomplete.